### DISCUSSION TITLE
大家都拿ChatGPT來做什麼？

### DISCUSSION THREAD

**Discussion by sjh501** [1101-GS4719-C/Python-Community-GS4719-C#1262] [General]
ChatGPT紅起來後看到室友在用ChatGPT查資料也跟風用了一下。一開始查一些生活上的小知識還不錯用，但發現有些問題它回答似是而非，與我之後在瀏覽器查到的資料不盡相同，而且也不是最近新的消息或知識。相同一些比較專業的問題也回答的模稜兩可、或者每次的生成答案都不相同、相牴觸的情況，因此也就不再用它了。只是放著這樣一個免費的工具不用又有些覺得可惜，所以想知道大家都拿ChatGPT來做什麼，感謝大家。

**Comment by brian431** [comment]
以資工系來說，大部份知識性的問題都能回答正確。另外很常詢問他一些Google不到的資訊也是很好用，例如問他一首英文歌有哪些相似的歌，或是問他電影裡面一些好奇的地方，或是叫他幫你生成報告的大綱及講稿等等，一天當中我問的問題不下十個，且大部分都能得到不錯的回答。

對於我們非母語人士來說還能很好地用來學習英文。像是問他一個句子有什麼不同的講法，或是問他你寫的句子有什麼可以改進的地方等等，任何的單字文法問題也是它所擅長的。想讓自己英文進步的話利用他是一個很好的方法，再加上用英文問的話通常效果會比較好建議還是多多用英文跟它溝通。

若是覺得他回答錯誤可以糾正一下或是讓問題描述更加清楚有時候就會給出正確答案了。

**Comment by DuxtinTai** [comment]
我認為chatgpt真的是大大的推進學習的效率，舉個最近的應用為例:
將Excel與資料庫軟體結合使用，例如Microsoft Access或Oracle。這樣可以將大量的數據從資料庫中提取出來，並使用Excel進行分析和視覺化。使用這種組合，可以更輕鬆地操作和管理大量的數據，並提高數據分析的準確性和效率。
可以透過問chatgpt的方式學習，就不用地毯式的自己找功能，或是到處爬文喔~

**Comment by CCello** [comment]
chatgpt真的是個很強大的工具,我不論是課業還是感興趣的東西,都會透過chatgpt幫我解決,雖然不是每個問題都可以準確地回答,但多少可以給我一些靈感或方向。

**Comment by KevinCheng42** [comment]
ChatGPT的出現加速的我們學習的效率，以前在搜尋時常常抓不到關鍵字，但自從ChatGPT出現後，只要多問幾個問題，它除了會回覆我們外，也會給我們問問題的方向，讓我們能更快找到解答～

**Comment by poweichen00** [comment]
Chatgpt 在對於寫程式方便快速很多，不但會提供簡易的程式碼，還會有註解，減少非常多查資料的時間。

**Comment by harrycoconut** [comment]
我都用chatgpt來寫英文報告

**Comment by tower20021202** [comment]
我都用chatgpt來寫心得把文章寫得更加深動

**Comment by Evening218** [comment]
我曾經使用chatgpt幫我完成影片心得報告，以它生成的內容為基礎再做修改。另外我最近看到比較有趣的用法是對chatgpt發洩不好的情緒，大概是因為罵它比較不會有罪惡感，但個人還是不推薦這樣做。建議大家每次問完問題都要記得跟chatgpt說謝謝喔~

**Comment by Chiuhsianglin** [comment]
我前陣子在讀資料通訊的時候，都會直接問chat GPT某個名詞是什麼或是某兩個名詞的差別，他讓我省掉了在google搜尋時需要自己統整資料的時間，雖然有些東西問chat gpt會得到錯誤的答案，但我還是認為它是一個很棒的工具

**Comment by pigeon9975** [comment]
我覺得chat gpt 除了在學業上是個很好用的工具，他可以快速你大方向，細項的部分再google確認資料正確性就好了，除此之外他還可以跟你玩迷宮、猜拳等遊戲，如果沒朋友孤單寂寞覺得冷，說不定AI是你的好選擇！

**Comment by ziihui** [comment]
Chatgpt是最棒的發明 my god🥹🔥
最近都用來學python！
因為完全沒學過程式，所以一開始上手有點吃力😢
但有了chatgpt後，他真的一步一步慢慢教我！省下很多時間，就不用到處請教別人😍
當我看不懂它的回答時，他還會換句話說～
再也不用害怕484問了笨問題～～～

**Comment by Jack-1D** [comment]
原本打算用它來幫我節省在Google上找資料的時間，這樣就可以減少debug的時間。但是後來發現，有很多細節是ChatGPT不會告訴我的。像是我在Linux上的Python code下SQL指令。在Windows可以執行的指令，到Linux就會報錯。我也問了ChatGPT可能錯在哪，他都只會改不同的方法回答，並沒有解決問題。最後我在Google上終於找到有篇文章在講Linux上很注重大小寫，才終於把問題解決了。後來我問ChatGPT是不是大小寫的問題，他才說他忘了跟我說...

**Comment by llh26** [comment]
剛剛去翻了我在ChatGPT的歷史紀錄，最近我大多用來問一些程式問題(python)、企業資料通訊的各名詞解釋，還有一些創意發想相關的問題（有時候靈感枯竭⋯）。我認為ChatGPT真的能大幅縮短我需要各方去查資料的時間，雖然有時候回覆內容會有點怪，但還是一個很不錯的工具！最近在使用上遇到一個小問題，我希望它用繁體中文回答我，它答應我後又開始用簡體中文了⋯，但講了兩三次後目前已有改善，跟各位分享～

**Comment by orianna901001** [comment]
我基本上都是拿來寫英文報告，查一些名詞之間的相關性，但這學期上機器學習的線上課程時，有看到有人用chatgtp寫出對話式遊戲，覺得沒事也可以來玩一下。
